orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
> I'll also note that it found an overflow in the sort_order field with one of my attempts to improve _update. This is the sort of edge-case that I would not have found with a simple test.

That sounds good, although I'm a bit surprised/alarmed that `sort_order` can overflow at all outside of a case where we've basically filled up the database, at which point we have other issues.  Theoretically any field with a limited size can overflow in that situation, so I don't know if there's any value in designing around it.

Is there a test for this?  What else did it find?  If I'm missing information about what you've actually done here, please post it either in the issue or in the code itself.